packages/metadata/src/Decorated/storage/fromMetadata/createFunction.ts

```diff
@@ -157,7 +157,11 @@ function extendHeadMeta (registry: Registry, { meta: { documentation, name, type
     type: registry.createType('StorageEntryTypeLatest', registry.createType('PlainTypeLatest', type.isMap ? type.asMap.key : type.asDoubleMap.key1), 0)
   });
 
-  return registry.createType('StorageKey', iterFn, { method, section });
+  const prefixKey = registry.createType('StorageKey', iterFn, { method, section });
+
+  return (arg?: any): StorageKey => !isUndefined(arg) && !isNull(arg)
+    ? registry.createType('StorageKey', iterFn(arg), { method, section })
+    : prefixKey;
```

Comment:
```suggestion
  return (arg?: any): StorageKey =>
    !isUndefined(arg) && !isNull(arg)
      ? registry.createType('StorageKey', iterFn(arg), { method, section })
      : prefixKey;
```

Just slight OCD, i.e. my eyes follow it better together.